src/ui/locales/fr.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Basculer l'attribution",
+    "AttributionControl.MapFeedback": "Commentaires sur la carte",
+    "FullscreenControl.Enter": "Passer en mode plein écran",
+    "FullscreenControl.Exit": "Quitter le mode plein écran",
+    "GeolocateControl.FindMyLocation": "Trouver ma position",
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
+    "LogoControl.Title": "Logo MapBox",
+    "Map.Title": "Carte",
+    "NavigationControl.ResetBearing": "Réinitialiser le relèvement vers le nord",
+    "NavigationControl.ZoomIn": "Agrandir",
+    "NavigationControl.ZoomOut": "Dézoomer",
+    "ScaleControl.Feet": "pi",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "kilomètres",
+    "ScaleControl.Miles": "moi",
```

Comment:
moi and pi? It should be ft and mi, right? In other languages there are also problems with wrong translations.

Reply:
Probabaly ft and mi or mile. I don't know of abbreviated french version of those units. pi would be literal. Why isn't kilometers represented by "km" though?

Reply:
Yep, it's the Google Translate mistakes